libraries/AP_GPS/AP_GPS_MAV.cpp

```diff
@@ -123,6 +123,9 @@ void AP_GPS_MAV::handle_msg(const mavlink_message_t &msg)
                 uint32_t corrected_ms = jitter.correct_offboard_timestamp_msec(timestamp_ms, now_ms);
                 state.last_corrected_gps_time_us = (corrected_ms * 1000ULL);
                 state.corrected_timestamp_updated = true;
+                if (state.last_corrected_gps_time_us) {
+                    _last_itow = state.time_week_ms;
+                }
```

Comment:
Could we get a comment in here as to why `last_corrected_gps_time_us` is a reasonable gate to us setting from `state.time_week_ms`?